src/api/status/public/js/build-log/build-header.js

```diff
@@ -29,6 +29,24 @@ function renderSha(compare, after) {
   buildGitSHA.innerText = after.substring(0, 7);
 }
 
+function renderLoadingAnimation(code) {
+  buildLoadingAnimation.class = 'material-icons';
+
+  switch (code) {
+    case 0:
```

Comment:
Nit: Document these if you can. 0, 1, and default seem like magic numbers.

Reply:
@Andrewnt219 I made some comments on top of it.